api/src/test/java/uk/gov/dhsc/htbhf/claimant/service/EligibilityAndEntitlementServiceTest.java

```diff
@@ -64,21 +74,30 @@
     @Mock
     EligibilityAndEntitlementDecisionFactory eligibilityAndEntitlementDecisionFactory;
 
-    @Test
-    void shouldReturnDuplicateWhenLiveClaimAlreadyExists() {
+    @ParameterizedTest
+    @MethodSource("allEligibilityOutcomeValues")
+    void shouldReturnDuplicateWhenLiveClaimAlreadyExists(EligibilityOutcome eligibilityOverrideOutcome) {
         //Given
         UUID existingClaimId = UUID.randomUUID();
         given(claimRepository.findLiveClaimWithNino(any())).willReturn(Optional.of(existingClaimId));
 
         //When
-        EligibilityAndEntitlementDecision decision = eligibilityAndEntitlementService.evaluateNewClaimant(CLAIMANT);
+        EligibilityAndEntitlementDecision decision = eligibilityAndEntitlementService.evaluateNewClaimant(CLAIMANT, eligibilityOverrideOutcome);
 
         //Then
         assertThat(decision.getEligibilityStatus()).isEqualTo(EligibilityStatus.DUPLICATE);
         assertThat(decision.getExistingClaimId()).isEqualTo(existingClaimId);
         verifyNoInteractions(client);
     }
 
+    private static Stream<EligibilityOutcome> allEligibilityOutcomeValues() {
```

Comment:
Technically this isn't all eligibility outcomes as we're not testing NOT_SET, do we want to add handling for this value? If not, we should probably rename this method accordingly, something like `allValidEligibilityOutcomes`.

Reply:
updated with all outcomes

Reply:
Should we use an EnumSource and another test for null?

Reply:
as discussed, I'm avoiding that because we're only testing one thing - that all values of EligibilityOutcome are ignored when there is a duplicate nino.